src/screens/CreateTokenAmount.js

```diff
@@ -141,7 +141,7 @@ class CreateTokenAmount extends React.Component {
                       {hathorLib.numberUtils.prettyValue(this.state.deposit)} HTR
                     </Strong></Text>,
                     <Text>
-                      {jt`You have ${amountAvailableText} HTR available`}
+                      {jt`You have ${amountAvailableText} available`}
```

Comment:
_Unrelated to line_

We should change the `amountAvailableText` to use the native token symbol we get from the fullnode.
This is because side-dags may use another symbol for the native token.

Reply:
Fixed on the `CreateTokenAmount` and `CreateTokenConfirm` screens. There are still other instances of hardcoded `HTR` messages in other places in the code, but those must be fixed in another PR.

![Fixed native token](https://github.com/user-attachments/assets/77394434-0389-4727-bfc2-e926b1fe9266)

Reply:
To reproduce this test, change inside the `src/sagas/wallet.js` -> `startWallet()`:
```js
    const serverInfo = yield call(wallet.start.bind(wallet), {
      pinCode: pin,
      password: pin,
    });

    serverInfo.native_token = {
      name: 'MyNative',
      symbol: 'NTV',
    };
    yield put(setServerInfo(serverInfo));
    console.log(`New serverInfo:`, serverInfo);
 ```